internal/testutil/acc/shared_resource.go

```diff
@@ -47,6 +48,8 @@ func ProjectIDExecution(tb testing.TB) string {
 		sharedInfo.projectName = RandomProjectName()
 		tb.Logf("Creating execution project: %s\n", sharedInfo.projectName)
 		sharedInfo.projectID = createProject(tb, sharedInfo.projectName)
+	} else {
+		time.Sleep(20 * time.Second) // HELP-65223: sleep a few seconds so clusters are not created concurrently in the execution project
```

Comment:
if we have 3 different parallel tests using ProjectIDExecution and creating a cluster, we still would have 2 of them being created at the same time right?
Maybe a adding a random would help, I understand this would be temporary to verify if concurrency is the underlying cause.

Reply:
no, this method is serialized with a mutex, so they'll be waiting 20s each

Reply:
in the beginning I tried 10s and worked, but failed with 5s, so at the moment I'm trying to 20s to be in the safe side to see if this is the issue with out of capacity errors

Reply:
do you mean 5 seconds still continued to result in OUT_OF_CAPACITY?

Reply:
yes 5s seems not be enough to avoid concurrency issues in Atlas, if that's the issue